src/rules.jl

```diff
@@ -170,29 +172,34 @@ end
     :NaN, :(  (SpecialFunctions.bessely($ν - 1, $x) - SpecialFunctions.bessely($ν + 1, $x)) / 2  )
 @define_diffrule SpecialFunctions.besselk(ν, x)   =
     :NaN, :( -(SpecialFunctions.besselk($ν - 1, $x) + SpecialFunctions.besselk($ν + 1, $x)) / 2  )
+@define_diffrule SpecialFunctions.besselkx(ν, x)  =
+    :NaN, :( -(SpecialFunctions.besselkx($ν - 1, $x) - 2SpecialFunctions.besselkx($ν, $x) + SpecialFunctions.besselkx($ν + 1, $x)) / 2 )
```

Comment:
Slightly simpler and consistent with SpecialFunctions (https://github.com/JuliaMath/SpecialFunctions.jl/blob/1febdd05902f5b3a3adc9915aba40b776aa3ea24/src/chainrules.jl#L109):
```suggestion
    :NaN, :( -(SpecialFunctions.besselkx($ν - 1, $x) + SpecialFunctions.besselkx($ν + 1, $x)) / 2 + SpecialFunctions.besselkx($ν, $x)  )
```